src/scss/components/optional/_menu.scss

```diff
@@ -168,17 +122,61 @@ $menu--expandable-linkActiveFontSize    : base--scaleUp !default;
             &:focus {
                 color: $menu-link-color;
             }
-        }
-            .c-menu-link:active,
-            .c-menu-link:focus,
-            .c-menu-link:hover,
+
+            &:hover,
+            &:active,
+            &:focus,
             .c-menu-link--active {
                 border-left: solid $menu-border-width--active $menu-border-color--active;
                 font-weight: $font-weight-bold;
                 margin-left: -1px;
             }
+        }
 
         .c-menu-icon {
             flex-shrink: 0;
         }
+
+        .c-menu--expandable {
+            @include media('<mid') {
+                background-color: $menu--expandable-bgColor;
+                border-bottom: solid 1px $menu--expandable-borderBottom;
+                box-shadow: 0 3px 4px 0 rgba(0, 0, 0, 0.12);
+                height: auto;
+
+                .c-menu-link {
+                    display: none;
+                    font-family: $font-family-headings;
+                    @include font-size($menu--expandable-linkFontSize);
+                    margin-left: 0;
+                    text-align: center;
+
+                    &,
+                    &:active,
+                    &:focus,
+                    &:hover {
+                        border: 0;
+                        font-weight: $font-weight-base;
+                    }
+                }
+
+                .c-menu-link--active {
+                    @include font-size($menu--expandable-linkActiveFontSize);
+                    text-decoration: underline;
+                    width: 100vw;
+                }
+            }
+        }
+
+        .c-menu--expandable--expanded {
```

Comment:
Should this be another indent-level in relative to `.c-menu--expandable`, for consistency with the namespace/indentation rules?

Reply:
It should, I'll do that in a following PR if that's OK 😸